### PR TITLE
updated contributing guide for template

### PR DESCRIPTION
**Pull request by KarishmaGhiya**
- update contributing guide for template with the link to root contributing guide and other details - CLA, Code of Conduct etc.

@azure/azure-sdk-eng

### CODE REVIEW

File: docs/README-TEMPLATE.md [hunk 1]
```diff
@@ -92,7 +92,19 @@ If the package or a related package supports it, include tips for logging or ena
 * If you think there's a good chance that developers might stumble across your package in error (because they're searching for specific functionality and mistakenly think the package provides that functionality), point them to the packages they might be looking for.
 
 ## Contributing
-Provide a link to the CONTRIBUTING.md of the repo.
+
+This project welcomes contributions and suggestions. Most contributions require you to agree to a Contributor
```
Comment (weshaggard) [inline docs/README-TEMPLATE.md]: Do we think it is worth duplicating the legal text in every library readme? I'd say no.

Comment (KarishmaGhiya) [reply docs/README-TEMPLATE.md]: ok

File: docs/README-TEMPLATE.md [hunk 2]
```diff
@@ -92,7 +92,8 @@ If the package or a related package supports it, include tips for logging or ena
 * If you think there's a good chance that developers might stumble across your package in error (because they're searching for specific functionality and mistakenly think the package provides that functionality), point them to the packages they might be looking for.
 
 ## Contributing
-Provide a link to the CONTRIBUTING.md of the repo.
+
+If you'd like to contribute to this library, please read the [contributing guide](https://github.com/Azure/azure-sdk-for-js/blob/master/CONTRIBUTING.md) to learn more about how to build and test the code.
```
Comment (weshaggard) [inline docs/README-TEMPLATE.md]: Please make this a relative link.

Comment (weshaggard) [reply docs/README-TEMPLATE.md]: even if it doesn't resolve in this repo make it relative `../../CONTRIBUTING.md`

Comment (KarishmaGhiya) [reply docs/README-TEMPLATE.md]: done

**Pull request merged**